test/patch/element.spec.js

```diff
@@ -114,17 +114,25 @@ describe('element', function () {
   });
 
   it('should have no effect while calling addEventListener without listener', function () {
+    var originAddEventListenerFn = zone.addEventListener;
```

Comment:
A nicer way to do that would be to fork the zone installing the spy at that time.

While your are modifying the test, please also check the args (`toHaveBeenCalledWith`)